axelrod/result_set.py

```diff
@@ -143,3 +147,51 @@ def csv(self):
                     for rank in self.ranking]
             writer.writerow(list(map(str, data)))
         return csv_string.getvalue()
+
+
+class ProbEndResultSet(ResultSet):
+    """A class to hold the results of a tournament."""
+
+    def __init__(self, players, prob_end, repetitions,
```

Comment:
Let me know if this is better (I'm sticking to "80 char PEP8..." but happy to change it).